test/dateTime.js

```diff
@@ -295,36 +295,36 @@ describe('dateTime', () => {
 		});
 
 		[
```

Comment:
To make it a bit easier to read, the order that these tests go through in each array is:
early/short/24-hour
early/short/12-hour
early/full/24-hour
early/full/12-hour
late/short/24-hour
late/short/12-hour
late/full/24-hour
late/full/12-hour